ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -85,9 +85,10 @@
 
 	private static final By ALLERGIES_LINK = By.id("allergyui-editAllergies");
 
-	private static final By CONDITIONS_LINK = By
-			.cssSelector(".conditions .info-header i.right");
+	private static final By CONDITIONS_LINK = By.cssSelector(".conditions .info-header i.right");
 
+	private static final By ATTACHMENTS_LINK = By.className("icon-share-alt edit-action right");
```

Comment:
So you may use attachment_Link under generalActions which is By id (**attachments.attachments.visitActions.default**)

Reply:
> So you may use attachment_Link under generalActions which is By id (**attachments.attachments.visitActions.default**)

Thanks @sherrif10 , to my implementation i would love to retrieve the element by its _className_. I can not see this attribute in the General/Action under Attachment. However am able to see the id attribute you are pointing at